datalad/distribution/siblings.py

```diff
@@ -0,0 +1,382 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Plumbing command for managing sibling configuration"""
+
+__docformat__ = 'restructuredtext'
+
+
+import logging
+
+from os.path import basename
+from os.path import relpath
+
+# XXX confusing: we have urljoin, _urljoin, dlurljoin
```

Comment:
Would be great to have an issue, at least outlining a way out of the misery -- or even a PR...

Thinking of proposing `_urldljoin_()`...